python-usfm-parser/usfm_grammar.py

```diff
@@ -280,184 +279,189 @@ def node_2_dict(node, usfm_bytes):
 title_query = USFM_LANGUAGE.query("""(title) @title""")
 
 class USFMParser():
-	"""Parser class with usfmstring, AST and methods for JSON convertions"""
+	"""Parser class with usfmstring, syntax_tree and methods for JSON convertions"""
 	def __init__(self, usfm_string):
 		# super(USFMParser, self).__init__()
 		self.USFM = usfm_string
-		self.USFMbytes = None
-		self.AST = None
+		self.USFM_bytes = None
```

Comment:
I think we should keep all the attribute names in lowercase for consistency sake.

Reply:
done